nengo_gui/components/netgraph.py

```diff
@@ -179,23 +170,20 @@ def _reload(self, code=None):
 
                 self.uids[uid] = new_item
 
-        self.to_be_expanded.append(model)
+        self.to_be_expanded.append(self.sim.model)
+
+        t_uids = [self.sim.get_uid(c.template) for c in self.sim.components]
```

Comment:
Is `t_uids` used anywhere below? (Maybe I just missed it because the function is so long)
What is the `t` supposed to stand for? Is there a more expressive name that could be used?